cachito/web/api_v1.py

```diff
@@ -72,6 +73,28 @@ def get_requests():
     # The call to `paginate` will inspect the current HTTP request for the
     # pagination parameters `page` and `per_page`.
     query = Request.query.order_by(Request.id.desc())
+    from_date = flask.request.args.get("from_date")
+    if from_date:
+        try:
+            from_date = date.fromisoformat(from_date)
+            from_date = datetime.combine(from_date, datetime.min.time())
+        except ValueError:
+            try:
+                from_date = datetime.fromisoformat(from_date)
```

Comment:
If you use `datetime.fromisoformat` in a date string, you have the same result as `date.fromisoformat` + `date.combine` you're using, so I guess you can simplify this code here.

```
>>> from_date = "2021-01-01"
>>> datetime.fromisoformat(from_date)
datetime.datetime(2021, 1, 1, 0, 0)
>>> datetime.combine(date.fromisoformat(from_date), datetime.min.time())
datetime.datetime(2021, 1, 1, 0, 0)
```

Reply:
Understood! My intention was to improve readability by writing extra code here to match the `to_date` code.